docs/src/developer/developer/how-to.md

```diff
@@ -16,15 +16,14 @@ Terminal 1:
 
 Terminal 2:
 * Compile all services: `make c`
-  * Note that you have to [import the public signing keys for nginx](https://github.com/wireapp/wire-server/blob/develop/services/nginz/README.md#common-problems-while-compiling) to be able to build nginz
-* Run services including nginz: `./services/start-services-only.sh`. If you don't want to run nginz set `INTEGRATION_USE_NGINZ=0`.
+* Run services including nginz: `./services/start-services-only.sh`.
 
 Open your browser at:
 
-- http://localhost:8080/api/swagger-ui for the swagger 2.0 endpoints (in development as of Feb 2021 - more endpoints will be added here as time goes on)
-- http://localhost:8080/swagger-ui/ for the old swagger 1.2 API (old swagger, endpoints will disappear from here (and become available in the previous link) as time progresses). Run `make -C services/nginz integration-test/conf/nginz/zwagger-ui` once to get JS libraries needed (they are not included in the repo).
+- `http://localhost:8080/api/swagger-ui` for the swagger 2.0 endpoints (in development as of Feb 2021 - more endpoints will be added here as time goes on)
```

Comment:
Nit: make them links instead of code blocks?

Reply:
d'oh :facepalm: